v2/client.go

```diff
@@ -744,6 +757,24 @@ func WriteBufferSize(writeBufferSize int) ClientOption {
 	}
 }
 
+// SetConnectionToLowLatencyServer is a ClientOption to set the client to connect
+// to server with lowest latency.
+func SetConnectionToLowLatencyServer() ClientOption {
```

Comment:
```suggestion
func SetConnectToLowLatencyServer() ClientOption {
```

To be consistent with `SetConnectToLowWorkLoadServer` below.